docs/application/dotnet/guides/nui/xaml/xaml-support-for-tizen-nui.md

```diff
@@ -0,0 +1,722 @@
+
+![XAMLCodeBehindFilePath](./media/xaml-page-xaml-cs-code-behind.PNG)
+
+> **Note**
+>
+> If you select other templates or work on lower version than Tizen 5.5, you have to manually create a C# code-behind file.
+
+In the following sample XAML code, the `x:Class` attribute specifies a fully qualified class name, the `XamlPage` class in the `NUIXamlTemplate` namespace. This means that this XAML file of **XamlPage.xaml.cs** defines a new class named `XamlPage` in the `NUIXamlTemplate` namespace that is derived from `ContentPage`:
+
+```csharp
+using System;
+using Tizen.NUI;
+using Tizen.NUI.BaseComponents;
+
+namespace NUIXamlTemplate
+{
+    public partial class XamlPage : ContentPage
+    {
+        public XamlPage(Window win) : base(win)
+        {
+            InitializeComponent();
+        }
+    }
+}
+```
+
+> **Note**
+>
+> The name of the XAML file must be same as the class name.
+
+### Load XAML
+
+To load layout instance using XAML, create a `XamlPage` instance in your code such as `OnCreate()` method that is invoked when the application is started:
+
+```csharp
+using System;
+using Tizen.NUI;
+using Tizen.NUI.BaseComponents;
+
+namespace NUIXamlTemplate
+{
+    class Program : NUIApplication
+    {
+        Animation animation;
+        protected override void OnCreate()
+        {
+            base.OnCreate();
+            Window.Instance.BackgroundColor = Color.Blue;
+            Window.Instance.KeyEvent += OnKeyEvent;
+
+            XamlPage page = new XamlPage(Window.Instance);
+
+            animation = new Animation(2000);
+            animation.AnimateTo(page.text, "Orientation", new Rotation(new Radian(new Degree(180.0f)), PositionAxis.X), 0, 500);
+            animation.AnimateTo(page.text, "Orientation", new Rotation(new Radian(new Degree(0.0f)), PositionAxis.X), 500, 1000);
+            animation.Looping = true;
+            animation.Play();
+        }
+
+        public void OnKeyEvent(object sender, Window.KeyEventArgs e)
+        {
+            if (e.Key.State == Key.StateType.Down && (e.Key.KeyPressedName == "XF86Back" || e.Key.KeyPressedName == "Escape"))
+            {
+                Exit();
+            }
+        }
+
+        static void Main(string[] args)
+        {
+            var app = new Program();
+            app.Run(args);
+        }
+    }
+}
+```
+
+When you compile and run this program, the `TextLabel` element appears on the window. Additionally, you can see the "Hello, NUI XAML APP" text, which is defined in the XAML file that rotates on the x-axis.
+
+## Create Solution for Tizen Version 5.0 and Lower
+
+To begin editing your first XAML file, use Visual Studio to create a new Tizen.NUI solution.
+
+In Visual Studio, select **File > New > Project** from the menu. In the **Create a new project** dialog that appears, select **C# > Tizen** from the selecting box, and then select **Blank App (Tizen.NUI)**.
```

Comment:
Please use numbered list and please verify:

To create a new project, follow these steps:

1. In Visual Studio, select **File > New > Project**.
2. In the **Create a new project** dialog that appears, select **C#** from the **All languages** drop-down list.
3. In the **All project types** drop-down list, select **Tizen**.
4. Select **Blank App (Tizen.NUI)** and click **Next**.